pilot/pkg/networking/core/v1alpha3/networkfilter.go

```diff
@@ -127,6 +128,8 @@ func buildOutboundNetworkFiltersStack(port *model.Port, tcpFilter *listener.Filt
 		filterstack = append(filterstack, buildOutboundMongoFilter(statPrefix))
 	case model.ProtocolRedis:
 		filterstack = append(filterstack, buildOutboundRedisFilter(statPrefix, clusterName))
+	case model.ProtocolMySQL:
+		filterstack = append(filterstack, buildOutboundMySQLFilter(statPrefix))
```

Comment:
you need a version guard here and in redis above as they didn't exist during 1.0 timeframe.

Reply:
Fixed in d9a7f20.